build/Makefile

```diff
@@ -32,7 +32,7 @@ base_version = 1.33.0
 #
 
 # Version defaults to the short hash of the latest commit
-VERSION ?= $(base_version)-$(shell git rev-parse --short=7 HEAD)
+VERSION ?= $(base_version)-dev-$(shell git rev-parse --short=7 HEAD)
```

Comment:
Good callout

Reply:
I'm going to change it back to `MAJOR.MINOR.PATCH-dev-HASH`, which means the part of `dev-HASH` is treated as pre-release version rather than build metadata. WDYT?

Reply:
Technically it's not semver, but it's only really used for our testing, so I don't think it actually matters 😄 

I say go for it.